deploy/eck-beats/examples/filebeat_no_autodiscover.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: beat.k8s.elastic.co/v1beta1
```

Comment:
This should be a values file not a custom resource

Reply:
This has been resolved.